CHANGELOG.md

```diff
@@ -7,7 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 ### Fixed
 - Fixed race condition during tests parallel execution (`tests::ingest::test_fetch::*` test group)
```

Comment:
Can you please also remove this line?

Reply:
Also I think this fix warrants a release

Reply:
Will do in the next PR

Reply:
Just for record, release PR: https://github.com/kamu-data/kamu-cli/pull/410